molecule/default/create.yml

```diff
@@ -4,7 +4,7 @@
   hosts: localhost
   connection: local
   gather_facts: false
-  no_log: "{{ not lookup('env', 'MOLECULE_DEBUG') | bool }}"
+  # no_log: "{{ not lookup('env', 'MOLECULE_DEBUG') | bool }}"
```

Comment:
Can you uncomment this? And the one below as well.

Reply:
updated